### PR TITLE
Nk single item

### PR DESCRIPTION
**Pull request by nashvegasnate**
Adding functionality to view a single item to purchase from the list of available items

### CODE REVIEW

File: client/src/components/ItemHeaderCard.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  Card, CardBody, CardTitle, Button
+} from 'reactstrap';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+function ItemHeaderCard({
+  itemID,
+  itemName
+}) {
+  const history = useHistory();
```
Comment (fimoefive) [inline client/src/components/ItemHeaderCard.js]: useHistory is imported, declared but never used.
Getting EsLint Error Codes.

File: client/src/views/SingleItemView.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        sellerID={item[0].sellerID}
+        categoryName={item[0].categoryName}
+        sellerFirstName={item[0].sellerFirstName}
+        sellerLastName={item[0].sellerLastName}
+        user={user}
+        setItems={setItems}
+        userFromDB={userFromDB}
+        />
+      }
+      {/* <ItemCard
```
Comment (fimoefive) [inline client/src/views/SingleItemView.js]: I would removed any commented out code that is not being used.

**Review by fimoefive — COMMENTED** [review]
I left two comments to your code. Single Item View is fully functional without errors. 
Great Job Nathan!

**Review by caseywalker — APPROVED** [review]
Looks good

**Pull request merged**